### PR TITLE
Bump rust-version to match actual MSRV

### PR DESCRIPTION
**Pull request by djc**
#85 bumped the MSRV but failed to bump the `rust-version` in `Cargo.toml`.

### CODE REVIEW

**Comment by djc** [comment]
CI failures are unrelated, just clippy lints.

**Comment by kchibisov** [comment]
Is clippy considering `rust-version` when running? As you can see, the CI with the same rust version is passing on current master, so it's related to your patch.

**Review by chrisduerr — CHANGES_REQUESTED** [review]
Can you change CI to use this script instead of relying on a manually set MSRV:

https://github.com/catacombing/catacomb/blob/master/.github/workflows/ci.yml#L21-L26

**Review by chrisduerr — APPROVED** [review]
Thanks

File: .builds/linux.yml
Comment (chrisduerr) [inline .builds/linux.yml:23]: This `cd` statement needs to be moved to the top.

**Review by chrisduerr — CHANGES_REQUESTED** [review]

**Comment by djc** [comment]
Fixed. As a point of feedback, as a drive-by contributor fixing up an oversight from an earlier PR, it doesn't feel great that you've blocked this PR on fixing up the CI flow (which then resulted in another round trip to deal with the error).

**Comment by kchibisov** [comment]
>Fixed. As a point of feedback, as a drive-by contributor fixing up an oversight from an earlier PR, it doesn't feel great that you've blocked this PR on fixing up the CI flow (which then resulted in another round trip to deal with the error).

There's no need to merge subpar code if we could have something more manageable. If you as a contributor doesn't want to do extra work that was explicitly requested, you're free to close your PR and we as a maintainers could pick it up. It's your choice in the end of the day whether you want to continue with the review or not. We don't ought each other anything in the end of the day.

**Comment by djc** [comment]
I just disagree that it's subpar -- it's strictly better than what it was before. You're asking me to implement an extra improvement that makes your life as maintainers easier. Anyway, I'm just giving feedback, up to you what you do with it.

**Comment by kchibisov** [comment]
>You're asking me to implement an extra improvement that makes your life as maintainers easier

I'm just saying that you could disagree with this at any point of time, so we implement it instead.

**Comment by chrisduerr** [comment]
> Fixed. As a point of feedback, as a drive-by contributor fixing up an oversight from an earlier PR, it doesn't feel great that you've blocked this PR on fixing up the CI flow (which then resulted in another round trip to deal with the error).

Considering I've linked you the exact bash line to fix it and that was absolutely trivial, I'm honestly shocked at how much time it took you.

If you can't figure out something so trivial, I'd recommend not pointlessly wasting maintainers' time on spoonfeeding you solutions and just skipping the PR submission entirely.

**Review by chrisduerr — APPROVED** [review]

**Pull request merged**
